dist/filelist

```diff
@@ -80,6 +80,7 @@ src/conn/conn_chunkcache.c
 src/conn/conn_ckpt.c
 src/conn/conn_compact.c
 src/conn/conn_dhandle.c
+src/conn/conn_evict.c
```

Comment:
Creating this gives us both `conn_cache.c` and `conn_evict.c`. Eviction is an action taken to service the cache. Do you intend to remove `conn_cache.c`? Or, at least to me, the "in which file do I put something new" question becomes potentially harder to reason about.